source/_docs/installation/recommended-hardware.markdown

```diff
@@ -0,0 +1,25 @@
+---
+title: "Recommended Hardware"
+description: "Recommended hardware for Home Assistant."
+---
+
+## minimum requirements
+
+Type | Minimum
+-- | --
+Storage | 32 GB
+Memory | 1 GB
+Network | 100 Mb/s wired
+Power (if Pi) | At least 2.5A
+
+## Performance expectations
+
+This is a list of popular platforms and what to expect from them.
+
+Platform | Notes
+-- | --
+Raspberry Pi Zero/Pi 2 | **Only** use these for testing
+Raspberry Pi 3/3+/4 | This is a good starting point, and depending on the amount of devices you integrate this can be enough - use an [A2 class SD](https://amzn.to/2X0Z2di) card if possible.
+NUC i3 | This is if you need a little more power over a Pi
+NUC i5 | This will allow you to run multiple services without any issues, perfect for a homelab
+NUC i7/i9 | Pure power, you should not have *any* performance issues
```

Comment:
I'm not sure why this is split out.

Making a lot of smaller pages, is really hard to follow, as everything is spread out everywhere. Instead, we should combine more (as we are doing already across the board) and provide navigation on the page itself.